README.md

```diff
@@ -63,7 +69,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+"Scope" references the ability to access a variable(s). If a variable is declared "globally," it is able to be accessed from anywhere within a program. If a variable is declared within a function, it is "local" and, therefore, can only be accessed within that function.
```

Comment:
Yes! I wanted to point out one rule though that you did not explicitly mention: every function and only functions create a new local scope. Additionally, scope does not just refer to variables, it includes functions and objects that you have access too as well.